examples/generateCatalog.py

```diff
@@ -0,0 +1,183 @@
+#!/usr/bin/env python
+# encoding: utf-8
+"""
+generate_star_grid_instance_catalog.py
+
+Make a PhoSim instance catalog with a grid of identical stars.
+
+Many things copied from the DESC ImageSimulationRecipes simGridStars.py
+
+Created by Michael Schneider on 2014-10-03
+"""
+
+import argparse
+import itertools
+import sys
+import math
+import random
+import os.path
+import subprocess, time
+import numpy as np
+import logging
+
+
+# Print log messages to screen:
+# logging.basicConfig(level=logging.DEBUG,
+#                     format='%(asctime)s - %(levelname)s - %(message)s')
+# Print log messages to file:
+logging.basicConfig(filename='log_generate_star_grid_instance_catalog.log',
+                    level=logging.DEBUG,
+                    format='%(asctime)s - %(levelname)s - %(message)s')
+
+
+###########################################################
+# Utilities to be exported to python libraries
+###########################################################
+def gen_star_list(outfilename, type, fov, ra_cen, dec_cen, nstars_per_dim, seeing, filter, seed, histid):
+	sed_filename = "../sky/sed_flat.txt"
```

Comment:
I'm assuming this is not an issue with the file.  There must be a bug in githubs rendering that makes it look like the indentation is not the same on line 37 as it is on line 38